recipes/slurm/build.sh

```diff
@@ -0,0 +1,4 @@
+./configure --prefix=$PREFIX --sysconfdir=$PREFIX
+make -j$CPU_COUNT
+make install
```

Comment:
Is there a `make check`?

Reply:
added. I'm not sure it does much of anything based on running the build locally, but it doesn't error. I guess it's good to have it.